scss/bootstrap/overrides/style/_forms.scss

```diff
@@ -75,6 +89,36 @@ label {
   }
 }
 
+// Style new group of type radio
+.form-inline {
+
+  &.form-inline-tabs .form-group {
+    box-shadow: $crm-form-control-shadow;
+    border: 1px solid $crm-darker-grayblue;
+    line-height: 1em;
+    padding: 8px 15px;
+    margin-right: -5px;
```

Comment:
Its only coming from inline. no margin in the fields. There is not common margin variable for this. Should i replace this with float left?